Dockerfile

```diff
@@ -21,3 +21,5 @@ COPY --from=builder /app/target/simpft-1.0.jar /app/app.jar
 ENTRYPOINT ["java", "-jar", "app.jar"]
 
 CMD ["--help"]
+
+VOLUME [ "/data" ]
```

Comment:
Do we need to specify the persistence in the dockerfile or is it the responsibility of the user to mount a volume to explicitly have persistence?

Reply:
Yes, we do need to specify the volume. Otherwise, docker won't know how to mount the specified volume into the container.